# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mono-develop/compile-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur monodevelop-0.9-orig/Core/src/MonoDevelop.Projects/MonoDevelop.Projects/Combine.cs monodevelop-0.9/Core/src/MonoDevelop.Projects/MonoDevelop.Projects/Combine.cs
--- monodevelop-0.9-orig/Core/src/MonoDevelop.Projects/MonoDevelop.Projects/Combine.cs	2005-11-04 12:01:53.000000000 +0000
+++ monodevelop-0.9/Core/src/MonoDevelop.Projects/MonoDevelop.Projects/Combine.cs	2006-01-27 17:57:14.029464750 +0000
@@ -17,7 +17,7 @@
 using System.Threading;
 
 using Mono.Unix.Native;
-using FileMode = Mono.Unix.FilePermissions;
+using FileMode = Mono.Unix.Native.FilePermissions;
 
 using MonoDevelop.Core;
 using MonoDevelop.Core.Properties;
